cluster-autoscaler/expander/grpcplugin/protos/expander.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package grpcplugin;
+import "k8s.io/api/core/v1/generated.proto";
+//import "google/protobuf/struct.proto";
+option go_package = "cluster-autoscaler/expander/grpcplugin/protos";
+
+
+
+// Interface for Expander
+service Expander {
+
+  rpc BestOptions (BestOptionsRequest)
+    returns (BestOptionsResponse) {}
+}
+
+message BestOptionsRequest {
+  repeated Option options = 1;
+  // key is node id from options
+  map<string, k8s.io.api.core.v1.Node> nodeInfoMap = 2;
```

Comment:
So over gRPC `nodeInfoMap` is really a Node map: is that required to save rpc calls throughput and simplify the proto?

Reply:
called out in readme, and renamed variable to be more appropriate